test/integration/install_simple/controls/server.rb

```diff
@@ -49,6 +49,18 @@
     it { should exist }
   end
 
+  describe file("/home/ubuntu/forseti-scripts/run_forseti.sh") do
+    it { should exist }
+  end
+
+  describe command("bash /home/ubuntu/forseti-scripts/run-forseti.sh") do
```

Comment:
The build is returning exit code 127 which means it cannot find the file. The script path is `run_forseti.sh`, but here it is `run-forseti.sh`.

Reply:
Done.